api_app/api/routes/airlock.py

```diff
@@ -164,13 +201,14 @@ async def create_review_user_resource(
         templateName=user_resource_template_name,
         properties={
             "display_name": "Airlock Review VM",
-            "description": f"Airlock Review VM for request {airlock_request.title} (ID {airlock_request.id})",
```

Comment:
Maybe keep the "Airlock Review VM" part there? Just so it's immediately obvious to eg TRE Admin, which VMs are Airlock Review VMs

Reply:
I removed this because it's already in the display_name, so that should be how they'd quickly identify? Then means they can quickly glance at the description for the request ID